packages/plugin-legacy/index.js

```diff
@@ -219,6 +219,9 @@ function viteLegacyPlugin(options = {}) {
       // @ts-ignore avoid esbuild transform on legacy chunks since it produces
       // legacy-unsafe code - e.g. rewriting object properties into shorthands
       opts.__vite_skip_esbuild__ = true
+      // @ts-ignore
+      // inject flag to prevent emit assets for legacy bundle
+      opts.__vite_legacy = true
```

Comment:
```suggestion
      opts.__vite_skip_asset_emit__ = true
```

- Adds `__` at the end to be consistent with `__vite_skip_esbuild__`
- Describes the actual consequence of the flag